en/resources/tutorials/swiftmetrics.md

```diff
@@ -51,7 +51,7 @@ and amend Package.swift to include the SwiftMetrics dependency with the major an
 
 ```swift
    dependencies: [
-      .Package(url: "https://github.com/RuntimeTools/SwiftMetrics.git", majorVersion: #, minorVersion: #)
+      .package(url: "https://github.com/RuntimeTools/SwiftMetrics.git", .upToNextMinor(from: "#.#.#"))
```

Comment:
In the generator I think we pin to majors of SwiftMetrics rather than minors? Shouldn't we do the same here?

Reply:
done